### PR TITLE
Pa/iteration 3 methods

### PR DESCRIPTION
**Pull request by palworth**
Minor changes in Stat_tracker to better suit OOP design.

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -17,5 +16,40 @@ def initialize(game_path, team_path, game_teams_path)
...
     @team_path = team_path
     @game_teams_path = game_teams_path
+    @game_collection = game_collection
+  end
+
+  def game_collection
+     GameCollection.new(@game_path)
+  end
+
+  def percentage_home_wins
```
Comment (palworth) [inline lib/stat_tracker.rb]: We are going to change these methods so they are referenced in the stat_tracker, but live in @game_collection

File: lib/team.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+require 'csv'
+require_relative 'team'
+
```
Comment (palworth) [inline lib/team.rb]: For team, we decided to use class methods.

File: test/fixtures/teams.csv [hunk 3]
```diff
@@ -3,3 +3,31 @@ team_id,franchiseId,teamName,abbreviation,Stadium,link
 4,16,Chicago Fire,CHI,SeatGeek Stadium,/api/v1/teams/4
 26,14,FC Cincinnati,CIN,Nippert Stadium,/api/v1/teams/26
 14,31,DC United,DC,Audi Field,/api/v1/teams/14
+6,6,FC Dallas,DAL,Toyota Stadium,/api/v1/teams/6
```
Comment (palworth) [inline test/fixtures/teams.csv]: needed more information/data for iteration 3 methods.

File: test/team_test.rb [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+require './lib/team'
+
+class TeamTest < Minitest::Test
+  def setup
+    @new_team = Team.new ({
+      :team_id => 1,
+      :franchiseId => 23,
+      :teamName => "Atlanta United",
+      :abbreviation => "ATL",
+      # :Stadium => "Mercedes-Benz Stadium"
```
Comment (palworth) [inline test/team_test.rb]: Not sure if we want to incorporate this. Also, need to change the symbol from Stadium to stadium so ruby doesnt think its a class if we do incorporate it

File: test/game_collection_test.rb [hunk 5]
```diff
@@ -63,9 +63,8 @@ def test_count_of_games_by_season_method
   def test_average_goals_per_game_method
     assert_equal 4.38, @game_collection.average_goals_per_game
   end
+  def test
```
Comment (DavidHoltkamp1) [inline test/game_collection_test.rb]: want this in teams test?

File: lib/game_collection.rb [hunk 6]
```diff
@@ -59,8 +59,16 @@ def count_of_games_by_season
   def average_goals_per_game
     game_goals_total = @games.sum {|game| game.away_goals + game.home_goals}
     (game_goals_total / @games.length.to_f).round(2)
+
+  end
+# 	Average number of goals scored in a game across all seasons including both home and away goals (rounded to the nearest 100th)	Float
+
+  def count_teams
+    @game_collection.team_id.uniq
```
Comment (DavidHoltkamp1) [inline lib/game_collection.rb]: want this in teams class ?

**Pull request merged**